pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -55,8 +57,11 @@ let
     homepage = "https://github.com/google/jax";
     license = licenses.asl20;
     maintainers = with maintainers; [ ndl ];
-    platforms = [ "x86_64-linux" "aarch64-darwin" "x86_64-darwin"];
-    hydraPlatforms = ["x86_64-linux" ]; # Don't think anybody is checking the darwin builds
+    platforms = platforms.unix;
+    # aarch64-darwin is broken because of https://github.com/bazelbuild/rules_cc/pull/136
+    # however even with that fix applied, it doesn't work for everyone:
+    # https://github.com/NixOS/nixpkgs/pull/184395#issuecomment-1207287129
+    broken = stdenv.isAarch64;
```

Comment:
Updated the meta to reflect the new state of platform support. Given that I got it working on `aarch64-darwin` (albeit only on my machine) I assume that not building on `aarch64-linux` is because something is broken, not because it's not supported.

Reply:
Maybe it even builds correctly, I haven't tried.